addons/web/static/src/views/fields/many2many_tags/many2many_tags_field.scss

```diff
@@ -6,10 +6,70 @@
     line-height: $o-line-height-base;
 }
 
-.o_field_widget.o_field_many2many_tags .o_input {
-    height: 100%;
+.o_field_widget.o_field_many2many_tags {
+    flex-flow: row wrap;
+
+    &.o_field_in_edition {
```

Comment:
what if the record is in edition but the field is readonly?

Reply:
it will have the class, but still be readonly, so I guess the style won't be correct